golem_sci/implementation.py

```diff
@@ -599,7 +599,14 @@ def _pull_eth_subscription_events(self) -> None:
                     self._on_event(t, sub.cb)
                 sub.last_pulled_block = self._confirmed_block
             except Exception as e:
-                logger.exception(
+                if (
+                        _is_jsonrpc_error(e)
+                        and 'missing trie node' in e.args[0].get('message')
```

Comment:
nit: `_is_jsonrpc_error` guarantees that `message` is present in `e.args[0]` so no need for `get` here
```suggestion
                        and 'missing trie node' in e.args[0]['message']
```